crates/cxx-qt-lib-headers/include/core/qcoreapplication.h

```diff
@@ -120,5 +120,12 @@ qapplicationSetOrganizationName(T& app, const QString& name)
   app.setOrganizationName(name);
 }
 
+template<typename T>
+void
+qapplicationRemoveLibraryPath(T& app, const QString& path)
+{
+  app.removeLibraryPath(path);
```

Comment:
in theory this could be `T::removeLibraryPath` as well but then `app` would be unused, so fine to do this :-D